crates/primitives/chain_config/src/rpc_version.rs

```diff
@@ -31,39 +26,42 @@ impl RpcVersion {
     }
 
     pub fn from_request_path(path: &str) -> Result<Self, RpcVersionError> {
-        log::debug!(target: "rpc_version", "extracting rpc version from request: {path}");
+        tracing::debug!(target: "rpc_version", "extracting rpc version from request: {path}");
 
         let path = path.to_ascii_lowercase();
-        let parts: Vec<&str> = path.split('/').collect();
-
-        log::debug!(target: "rpc_version", "version parts are: {parts:?}");
+        let parts: Vec<&str> = path.split('/').filter(|part| !part.is_empty()).collect();
```

Comment:
Does't filtering with `!part.is_empty()` mean something like `///rpc////v0_8_0////` would be counted as valid? I guess this is ok but it feels a little strange.

Reply:
for me it remains a valid path